src/Symfony/Bundle/FrameworkBundle/Resources/config/asset_mapper.php

```diff
@@ -75,6 +75,7 @@
                 param('kernel.project_dir'),
                 abstract_arg('array of excluded path patterns'),
                 abstract_arg('exclude dot files'),
+                abstract_arg('strict mode'),
```

Comment:
```suggestion
                param('kernel.debug'),
```

Reply:
Yup, Nicolas is right - I gave too complex of a recommendation. Do this, then no change needed in `FrameworkExtension`.

Reply:
Done!